src/Modal/Modal.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import ModalHeader from "./ModalHeader";
+import ModalActions from "./ModalActions";
+import Portal from "./Portal";
+
+class Modal extends Component {
+  state = {
+    open: false,
+  };
+
+  openModal = () => {
+    const { onOpen } = this.props;
+    this.setState({
+      open: true,
+    });
+
+    if (onOpen) {
+      onOpen();
+    }
+  };
+
+  closeModal = () => {
+    const { onClose } = this.props;
+    this.setState({
+      open: false,
+    });
+
+    if (onClose) {
+      onClose();
+    }
+  };
+
+  render() {
+    const {
+      className,
+      trigger,
+      hideCloseIcon,
+      title,
+      onConfirmClick,
+      confirmLabel,
+      cancelLabel,
+      children,
+    } = this.props;
+    const { open } = this.state;
+
+    return (
+      <div className={className}>
```

Comment:
What's the reason for wrapping it in a `div`? `className` should be applied to the modal dialog